### PR TITLE
Initial version with dist

### PR DESCRIPTION
**Pull request by henrahmagix**
@incuna/frontend @incuna/js Apologies for not making this into smaller pull-requests. Please merge, ta!

Ignore the `dist/` folder. Can't find a way to tell GitHub to not render the diff of some committed files in pull-requests, like it automatically does with `*.min.*` files.

### CODE REVIEW

**Comment by meshy** [comment]
`\o/`

File: src/angular-articles.js [hunk 1]
```diff
@@ -3,11 +3,181 @@
...
+                    controller: 'SectionsArticlesListCtrl',
+                    templateUrl: 'templates/sections/articles-list.html'
+                })
+                .when('/sections/:section/articles/:article/', {
+                    controller: 'SectionsArticlesDetailCtrl',
+                    templateUrl: 'templates/sections/articles-detail.html'
+                })
+                .otherwise({
+                    redirectTo: '/articles/'
+                });
```
Comment (perry) [inline src/angular-articles.js:40]: This will catch all other routes, probably don't want to do that.

Comment (henrahmagix) [inline src/angular-articles.js:40]: Done in 4a8e129

File: Gruntfile.js [hunk 2]
```diff
@@ -11,9 +11,227 @@ module.exports = function (grunt) {
...
+                logConcurrentOutput: true
+            },
+            watch: ['watch', 'karma:watch']
+        },
+
+        concat: {
+            options: {
+                stripBanners: true,
+                banner: '/*! <%= pkg.name %> - v<%= pkg.version %> - ' +
+                    '<%= grunt.template.today("yyyy-mm-dd") %> */\n'
```
Comment (perry) [inline Gruntfile.js]: Perhaps put a timestamp in there as well

Comment (henrahmagix) [inline Gruntfile.js]: Done in 7a038b8

File: dist/angular-articles.js [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+        function ($routeProvider) {
+            $routeProvider
+                .when('/articles/', {
+                    controller: 'ArticlesListCtrl',
+                    templateUrl: 'templates/articles/list.html'
+                })
+                .when('/articles/:article/', {
+                    controller: 'ArticlesDetailCtrl',
+                    templateUrl: 'templates/articles/detail.html'
+                })
```
Comment (perry) [inline dist/angular-articles.js:20]: What are these views used for if there are nested ones below?

Comment (henrahmagix) [inline dist/angular-articles.js:20]: An individual article without context of a section.

**Pull request merged**